mlflow/store/file_store.py

```diff
@@ -332,12 +332,7 @@ def _get_param_from_file(parent_path, param_name):
     def _get_tag_from_file(parent_path, tag_name):
         _validate_tag_name(tag_name)
         tag_data = read_file(parent_path, tag_name)
-        if len(tag_data) == 0:
-            raise Exception("Tag '%s' is malformed. No data found." % tag_name)
-        if len(tag_data) > 1:
-            raise Exception("Unexpected data for tag '%s'. Tag recorded more than once"
-                            % tag_name)
-        return RunTag(tag_name, str(tag_data[0].strip()))
+        return RunTag(tag_name, str(tag_data.strip()))
```

Comment:
Yeah I agree, there's nothing on my end that necessitates stripping, and it would make more sense if tag data weren't stripped in the context of this PR

Reply:
I went ahead and removed the call to `trim`

Reply:
I also had to modify `set_tag` in `FileStore` to not add a trailing newline (since otherwise the tests failed because there was an unexpected trailing newline that was not stripped out). I'm happy to revert these I don't care that much.

Reply:
Also worth noting that these changes are not backwards-compatible w.r.t. how existing notes would be displayed. I'm not sure what your policy is about that during alpha.